mobile/src/components/share/ScreenOption.jsx

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
is this considered a react component as well? we can move it to utilites/ folder otherwise and use .js

Reply:
Also fixed with 3d982e3f6932c7a54ab65afaa4fc41f4f35ca8ff